modular_skyrat/modules/opposing_force/code/equipment/misc.dm

```diff
@@ -81,4 +81,4 @@
 	description = "A dusty SUPER-SIZED crate from the back of the Syndicate warehouse. Rumored to contain a valuable assortment of items, \
 			but you never know. Contents are sorted to always be worth 125 TC."
 	telecrystal_count = 125
-	admin_note = "WARNING: Extremely high TC count, absolutely no guarantee whatsoever of what will come out of this. Contents equal to 125 TC."
+	admin_note = "WARNING: This should only be given out for events, absolutely no guarantee whatsoever of what will come out of this, from both nukie and traitor uplinks. Contents equal to 125 TC."
```

Comment:
```suggestion
	admin_note = "WARNING: This should only be given out for events, absolutely no guarantee whatsoever of what will come out of this. Contents equal to 125 TC."
```
it doesn't give ops gear

Reply:
word; I got an elite suit when I was testing it and assumed it was up to tomfoolery

Reply:
An official "tg powercrept elite suits to regular traitors" moment